server/php-runner.cpp

```diff
@@ -436,6 +436,9 @@ void PhpScript::run() noexcept {
   }
   assert (run_main->run != nullptr);
 
+  if (sigsetjmp(timeout_handler, true) != 0) { // set up a timeout recovery point for initialising
```

Comment:
That doesn't solve the problem completely. What if timeout is raised the line above?
I think that script timeout must be set strictly after this `sigsetjmp`